src/main/groovy/pl/allegro/tech/build/axion/release/util/VersionComparator.groovy

```diff
@@ -0,0 +1,141 @@
+package pl.allegro.tech.build.axion.release.util
+
+import java.util.Comparator
+
+class VersionComparator implements Comparator<String> {
```

Comment:
this probably can be deleted - `Version` is comparable and has great comparators